socorro/unittest/external/es/test_supersearch.py

```diff
@@ -561,6 +561,16 @@ def test_get_with_facets(self):
             _facets=['unkownfield']
         )
 
+    def test_get_missing_index(self):
```

Comment:
How about ``test_get_against_missing_index``? I don't like the words "non-existent". :-)